docs/usage/configuration-options.md

```diff
@@ -98,6 +98,19 @@ Currently it is needed/supported for the `helm-requirements` manager only.
 }
 ```
 
+The following `alias` will result in Renovate treating docker images prefixed with `registry.example.com/proxy-cache` as if they were prefixed with `docker.io` instead.
+Renovate will get the tag list from Docker Hub instead of `registry.example.com`.
+This is useful when using a proxy registry like [Harbor Proxy Cache](https://goharbor.io/docs/2.1.0/administration/configure-proxy-cache/) or [Gitlab Dependency Proxy](https://docs.gitlab.com/ce/user/packages/dependency_proxy/).
+These proxies do not support the endpoint for listing all tags and thus can not be used to discover new versions of images.
```

Comment:
```suggestion
These proxies do not natively support a endpoint that lists all tags and thus can not be used without this workaround to discover new versions of images.
```

Maybe a text like this is easier to understand?

Reply:
I applied this slightly changed in 31f596395d9f088f3d85f04900f8907ef0944b2c

Reply:
Thank you for the grammar fixes! 😄